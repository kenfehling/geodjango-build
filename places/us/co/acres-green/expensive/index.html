
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Is Acres Green expensive?</title>
    <meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=yes,minimum-scale=0.25,maximum-scale=5">
    <link rel="stylesheet" type="text/css" href="/static/style.css">
    <link rel="shortcut icon" href="/static/favicon/favicon.ico">
    <link rel="icon" sizes="16x16 32x32 64x64" href="/static/favicon/favicon.ico">
    <link rel="icon" type="image/png" sizes="196x196" href="/static/favicon/favicon-192.png">
    <link rel="icon" type="image/png" sizes="160x160" href="/static/favicon/favicon-160.png">
    <link rel="icon" type="image/png" sizes="96x96" href="/static/favicon/favicon-96.png">
    <link rel="icon" type="image/png" sizes="64x64" href="/static/favicon/favicon-64.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/static/favicon/favicon-32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/static/favicon/favicon-16.png">
    <link rel="apple-touch-icon" href="/static/favicon/favicon-57.png">
    <link rel="apple-touch-icon" sizes="114x114" href="/static/favicon/favicon-114.png">
    <link rel="apple-touch-icon" sizes="72x72" href="/static/favicon/favicon-72.png">
    <link rel="apple-touch-icon" sizes="144x144" href="/static/favicon/favicon-144.png">
    <link rel="apple-touch-icon" sizes="60x60" href="/static/favicon/favicon-60.png">
    <link rel="apple-touch-icon" sizes="120x120" href="/static/favicon/favicon-120.png">
    <link rel="apple-touch-icon" sizes="76x76" href="/static/favicon/favicon-76.png">
    <link rel="apple-touch-icon" sizes="152x152" href="/static/favicon/favicon-152.png">
    <link rel="apple-touch-icon" sizes="180x180" href="//static/favicon/favicon-180.png">
    <meta name="msapplication-TileColor" content="#FFFFFF">
    <meta name="msapplication-TileImage" content="//static/favicon/favicon-144.png">
    <meta name="msapplication-config" content="/static/favicon/browserconfig.xml">
    
    <meta name="keywords" content="Acres Green, Colorado, CO, expensive, housing, cost, price, statistics, location, places, real estate">
    <meta name="description" content="The median home in Acres Green costs $351,000. In comparison to its nearby areas, Acres Green is a little less expensive, and it&#x27;s a little more expensive compared to the entire country. It&#x27;s somewhat less expensive compared to the city of Denver.">
    <meta name="og:description" content="The median home in Acres Green costs $351,000. In comparison to its nearby areas, Acres Green is a little less expensive, and it&#x27;s a little more expensive compared to the entire country. It&#x27;s somewhat less expensive compared to the city of Denver.">
    
        <script type="application/ld+json">{"@context": "https://schema.org", "@type": "Article", "author": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "publisher": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "datePublished": "2021-07-11T09:40:47.301301", "dateModified": "2021-07-11T09:40:47.301301", "image": "https://storage.googleapis.com/statistipedia/img/maps/png/co/acres-green.png", "headline": "Is Acres Green expensive?", "mainEntity": {"@type": "City", "name": "Acres Green", "sameAs": "https://www.wikidata.org/wiki/Q1868131", "containedInPlace": {"@type": "State", "name": "Acres Green", "sameAs": "https://www.wikidata.org/wiki/Q1261"}}}</script>
    

    <meta name="og:locale" content="en_US" />
</head>

<body>
<header>
    <div class="inner-header">
        <a class="site-title" href="/">
            Statistipedia
        </a>
    </div>
</header>

<main>
    <article>
        <h1>Is Acres Green expensive?</h1>
        

<table class="infobox">
    <tr>
        <th colspan="2" class="title">Acres Green, CO</th>
    </tr>
    <tr>
        <td colspan="2" class="place-image">
            <picture>
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/webp/co/acres-green.webp" type="image/webp" />
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/png/co/acres-green.png" type="image/png" />
    <img src="https://storage.googleapis.com/statistipedia/img/maps/png/co/acres-green.png" loading="lazy" alt="Map of Acres Green" />
</picture>
        </td>
    </tr>
    <tbody class="main-info">
        <tr>
            <th>State</th>
            <td>
                <a href="/places/us/co">Colorado</a>
            </td>
        </tr>
        <tr>
            <th>Population</th>
            <td>3,100</td>
        </tr>
    </tbody>
    <tr>
        <th colspan="2" class="header">Links</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                
                
                    <li>
                        <a href="https://en.wikipedia.org/wiki/Acres_Green,_Colorado" target="_blank" rel="nofollow">
                            Wikipedia article
                        </a>
                    </li>
                
                <li>
                    <a href="https://data.census.gov/cedsci/profile?g=1600000US0800320" target="_blank">
                        Census profile
                    </a>
                </li>
            </ul>
        </td>
    </tr>
    <tr>
        <th colspan="2" class="header">Maps</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                <li><a target="_blank" href="https://www.openstreetmap.org/search?query=Acres Green%2C%20CO">OpenStreetMaps</a></li>
                <li><a target="_blank" href="https://www.google.com/maps/place/Acres Green,+CO">Google Maps</a></li>
                <li><a target="_blank" href="https://www.bing.com/maps?q=Acres Green,+CO">Bing Maps</a></li>
                <li><a target="_blank" href="https://www.mapquest.com/search/result?query=Acres Green,+CO">MapQuest</a></li>
            </ul>
        </td>
    </tr>
</table>
        
            <p class="lede">The median home in <a href="/places/us/co/acres-green">Acres Green</a> costs $351,000.</p>
            <p class="lede">In comparison to its nearby areas, Acres Green is a little less expensive, and it's a little more expensive compared to the entire country. It's somewhat less expensive compared to the city of <a href="/places/us/co/denver/expensive?">Denver</a>.</p>
            <div class="figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <div class="viz">
                
                    <div class="neighbor bar"
                         style="width: 100.0%">
                        Lone Tree
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 95.7570977917981%">
                        Inverness
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 75.66246056782335%">
                        Meridian
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 68.42271293375394%">
                        Centennial
                    </div>
                
                    <div class="city bar"
                         style="width: 61.60883280757098%">
                        Denver
                    </div>
                
                    <div class="location bar"
                         style="width: 55.36277602523659%">
                        Acres Green
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 48.12302839116719%">
                        Dove Valley
                    </div>
                
                    <div class="country bar"
                         style="width: 47.3186119873817%">
                        US average
                    </div>
                
            </div>
        </div>
        <figcaption>Comparison of median Acres Green home price to nearby areas</figcaption>
    </figure>
</div>
            
<div class="distro figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <img src="https://storage.googleapis.com/statistipedia/img/distro/co/acres-green.svg"
                 loading="lazy"
                 alt="Distribution of home prices" />
        </div>
        <figcaption>Distribution of home prices</figcaption>
    </figure>
</div>
            <h2>Nearby places</h2><h3>Lone Tree</h3><p>The city of <a href="/places/us/co/lone-tree/expensive?">Lone Tree</a> is Acres Green's neighbor to the southeast. Housing prices there are a lot more expensive. Its median home value is $634,000, which is $283,000 (80%) higher than Acres Green.</p><h3>Inverness</h3><p><a href="/places/us/co/inverness/expensive?">Inverness</a> is 2 miles northeast of Acres Green. Housing prices there are a lot more expensive. The median home price there is $607,100, which is $256,100 (72%) higher than Acres Green.</p><h3>Meridian</h3><p><a href="/places/us/co/meridian/expensive?">Meridian</a> is 2 miles southeast of Acres Green. The housing prices there are significantly more expensive. Its median home cost is $479,700, which comes out to $128,700 (36%) higher than Acres Green.</p><h3>Centennial</h3><p>The city of <a href="/places/us/co/centennial/expensive?">Centennial</a> is 2 miles northeast of Acres Green. The housing prices there are a bit more expensive. Its median home value is $433,800, which works out to $82,800 (23%) higher than Acres Green.</p><h3>Dove Valley</h3><p><a href="/places/us/co/dove-valley/expensive?">Dove Valley</a> is 3 miles east of Acres Green. Its real estate prices are a bit less expensive. Its median home price is $305,100, which comes out to $45,900 (13%) lower than Acres Green.</p>
            <h2>See also</h2>
<ul><li><a href="/places/us/co/highlands-ranch/expensive?">Highlands Ranch</a></li>
<li><a href="/places/us/co/littleton/expensive?">Littleton</a></li>
<li><a href="/places/us/co/cherry-hills-village/expensive?">Cherry Hills Village</a></li>
<li><a href="/places/us/co/stonegate/expensive?">Stonegate</a></li>
<li><a href="/places/us/co/foxfield/expensive?">Foxfield</a></li>
<li><a href="/places/us/co/cherry-creek/expensive?">Cherry Creek</a></li>
<li><a href="/places/us/co/greenwood-village/expensive?">Greenwood Village</a></li></ul>
        
        <nav class="categories">
            Categories:
            &nbsp;<a href="/places/us">United States</a>
            &nbsp;<span class="divider">|</span>
            &nbsp;<a href="/places/us/co">Colorado</a>
        </nav>
    </article>
</main>

<footer>
    <div class="inner-footer">
        <a class="site-title" href="/">
            Statistipedia
        </a>
        <div>

        </div>
        <ul>
            <li>© 2021 Statistipedia</li>
            <li><a href="/about">About</a></li>
            <li><a href="/privacy">Privacy</a></li>
        </ul>
    </div>
</footer>
</body>
</html>