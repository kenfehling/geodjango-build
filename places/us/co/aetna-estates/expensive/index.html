
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Is Aetna Estates expensive?</title>
    <meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=yes,minimum-scale=0.25,maximum-scale=5">
    <link rel="stylesheet" type="text/css" href="/static/style.css">
    <link rel="shortcut icon" href="/static/favicon/favicon.ico">
    <link rel="icon" sizes="16x16 32x32 64x64" href="/static/favicon/favicon.ico">
    <link rel="icon" type="image/png" sizes="196x196" href="/static/favicon/favicon-192.png">
    <link rel="icon" type="image/png" sizes="160x160" href="/static/favicon/favicon-160.png">
    <link rel="icon" type="image/png" sizes="96x96" href="/static/favicon/favicon-96.png">
    <link rel="icon" type="image/png" sizes="64x64" href="/static/favicon/favicon-64.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/static/favicon/favicon-32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/static/favicon/favicon-16.png">
    <link rel="apple-touch-icon" href="/static/favicon/favicon-57.png">
    <link rel="apple-touch-icon" sizes="114x114" href="/static/favicon/favicon-114.png">
    <link rel="apple-touch-icon" sizes="72x72" href="/static/favicon/favicon-72.png">
    <link rel="apple-touch-icon" sizes="144x144" href="/static/favicon/favicon-144.png">
    <link rel="apple-touch-icon" sizes="60x60" href="/static/favicon/favicon-60.png">
    <link rel="apple-touch-icon" sizes="120x120" href="/static/favicon/favicon-120.png">
    <link rel="apple-touch-icon" sizes="76x76" href="/static/favicon/favicon-76.png">
    <link rel="apple-touch-icon" sizes="152x152" href="/static/favicon/favicon-152.png">
    <link rel="apple-touch-icon" sizes="180x180" href="//static/favicon/favicon-180.png">
    <meta name="msapplication-TileColor" content="#FFFFFF">
    <meta name="msapplication-TileImage" content="//static/favicon/favicon-144.png">
    <meta name="msapplication-config" content="/static/favicon/browserconfig.xml">
    
    <meta name="keywords" content="Aetna Estates, Colorado, CO, expensive, housing, cost, price, statistics, location, places, real estate">
    <meta name="description" content="The price of the average home in &lt;a href=&quot;/places/us/co/aetna-estates&quot;&gt;Aetna Estates&lt;/a&gt; is $48,700.">
    <meta name="og:description" content="The price of the average home in &lt;a href=&quot;/places/us/co/aetna-estates&quot;&gt;Aetna Estates&lt;/a&gt; is $48,700.">
    
        <script type="application/ld+json">{"@context": "https://schema.org", "@type": "Article", "author": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "publisher": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "datePublished": "2021-07-08T18:17:12.013159", "dateModified": "2021-07-08T18:17:12.013159", "image": "https://storage.googleapis.com/statistipedia/img/maps/png/co/aetna-estates.png", "headline": "Is Aetna Estates expensive?", "mainEntity": {"@type": "City", "name": "Aetna Estates", "sameAs": "https://www.wikidata.org/wiki/Q4688748", "containedInPlace": {"@type": "State", "name": "Aetna Estates", "sameAs": "https://www.wikidata.org/wiki/Q1261"}}}</script>
    

    <meta name="og:locale" content="en_US" />
</head>

<body>
<header>
    <div class="inner-header">
        <a class="site-title" href="/">
            Statistipedia
        </a>
    </div>
</header>
<main>
<main>
    <article>
        <h1>Is Aetna Estates expensive?</h1>
        

<table class="infobox">
    <tr>
        <th colspan="2" class="title">Aetna Estates, CO</th>
    </tr>
    <tr>
        <td colspan="2" class="place-image">
            <picture>
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/webp/co/aetna-estates.webp" type="image/webp" />
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/png/co/aetna-estates.png" type="image/png" />
    <img src="https://storage.googleapis.com/statistipedia/img/maps/png/co/aetna-estates.png" alt="Map of Aetna Estates" />
</picture>
        </td>
    </tr>
    <tbody class="main-info">
        <tr>
            <th>State</th>
            <td>
                <a href="/places/us/co">Colorado</a>
            </td>
        </tr>
        <tr>
            <th>Population</th>
            <td>1,398</td>
        </tr>
    </tbody>
    <tr>
        <th colspan="2" class="header">Links</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                
                
                    <li>
                        <a href="https://en.wikipedia.org/wiki/Aetna_Estates,_Colorado" target="_blank" rel="nofollow">
                            Wikipedia article
                        </a>
                    </li>
                
                <li>
                    <a href="https://data.census.gov/cedsci/profile?g=1600000US0800620" target="_blank">
                        Census profile
                    </a>
                </li>
            </ul>
        </td>
    </tr>
    <tr>
        <th colspan="2" class="header">Maps</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                <li><a target="_blank" href="https://www.openstreetmap.org/search?query=Aetna Estates%2C%20CO">OpenStreetMaps</a></li>
                <li><a target="_blank" href="https://www.google.com/maps/place/Aetna Estates,+CO">Google Maps</a></li>
                <li><a target="_blank" href="https://www.bing.com/maps?q=Aetna Estates,+CO">Bing Maps</a></li>
                <li><a target="_blank" href="https://www.mapquest.com/search/result?query=Aetna Estates,+CO">MapQuest</a></li>
            </ul>
        </td>
    </tr>
</table>
        
            <p class="lede">The price of the average home in <a href="/places/us/co/aetna-estates">Aetna Estates</a> is $48,700.</p>
            <p class="lede">Aetna Estates is a lot less expensive compared to nearby areas and the US as a whole. Compared to <a href="/places/us/co/denver/expensive?">Denver</a> it's substantially less expensive.</p>
            <div class="figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <div class="viz">
                
                    <div class="neighbor bar"
                         style="width: 38.44015980545423%">
                        Foxfield
                    </div>
                
                    <div class="city bar"
                         style="width: 22.615945805106826%">
                        Denver
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 18.533958659023796%">
                        Commerce City
                    </div>
                
                    <div class="country bar"
                         style="width: 17.37015806843842%">
                        US average
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 16.791152799490476%">
                        Aurora
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 15.725783104626252%">
                        Bennett
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 15.343639627120607%">
                        Glendale
                    </div>
                
                    <div class="location bar"
                         style="width: 2.819755659776504%">
                        Aetna Estates
                    </div>
                
            </div>
        </div>
        <figcaption>Comparison of average Aetna Estates home price to nearby areas</figcaption>
    </figure>
</div>
            <h2>Nearby places</h2><h3>Foxfield</h3><p><a href="/places/us/co/foxfield/expensive?">Foxfield</a> is 11 miles southwest of Aetna Estates. The housing prices there are substantially more expensive. Its average home cost is $663,900, which is $615,200 (1263%) higher than Aetna Estates.</p><h3>Commerce City</h3><p><a href="/places/us/co/commerce-city/expensive?">Commerce City</a> is 11 miles northwest of Aetna Estates. Housing prices there are significantly more expensive. The average home value there is $320,100, which works out to $271,400 (557%) higher than Aetna Estates.</p><h3>Aurora</h3><p>The city of <a href="/places/us/co/aurora/expensive?">Aurora</a> is Aetna Estates's neighbor to the southwest. Its real estate prices are significantly more expensive. The average home price there is $290,000, which comes out to $241,300 (495%) higher than Aetna Estates.</p><h3>Bennett</h3><p><a href="/places/us/co/bennett/expensive?">Bennett</a> is 11 miles east of Aetna Estates. Housing prices there are substantially more expensive. Its average home cost is $271,600, which works out to $222,900 (457%) higher than Aetna Estates.</p><h3>Glendale</h3><p>The city of <a href="/places/us/co/glendale/expensive?">Glendale</a> is 12 miles west of Aetna Estates. The housing prices there are a lot more expensive. The average home price there is $265,000, which comes out to $216,300 (444%) higher than Aetna Estates.</p>
            <h2>See also</h2>
<ul><li><a href="/places/us/co/cherry-creek/expensive?">Cherry Creek</a></li>
<li><a href="/places/us/co/cherry-hills-village/expensive?">Cherry Hills Village</a></li>
<li><a href="/places/us/co/greenwood-village/expensive?">Greenwood Village</a></li>
<li><a href="/places/us/co/dove-valley/expensive?">Dove Valley</a></li>
<li><a href="/places/us/co/inverness/expensive?">Inverness</a></li>
<li><a href="/places/us/co/derby/expensive?">Derby</a></li>
<li><a href="/places/us/co/centennial/expensive?">Centennial</a></li></ul>
        
        <nav class="categories">
            Categories:
            &nbsp;<a href="/places/us">United States</a>
            &nbsp;<span class="divider">|</span>
            &nbsp;<a href="/places/us/co">Colorado</a>
        </nav>
    </article>
</main>
</main>
<footer>
    <div class="inner-footer">
        <a class="site-title" href="/">
            Statistipedia
        </a>
        <div>

        </div>
        <ul>
            <li>© 2021 Statistipedia</li>
            <li><a href="/about">About</a></li>
            <li><a href="/privacy">Privacy</a></li>
        </ul>
    </div>
</footer>
</body>
</html>