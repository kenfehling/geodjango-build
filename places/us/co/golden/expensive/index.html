
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Is Golden expensive?</title>
    <meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=yes,minimum-scale=0.25,maximum-scale=5">
    <link rel="stylesheet" type="text/css" href="/static/style.css">
    <link rel="shortcut icon" href="/static/favicon/favicon.ico">
    <link rel="icon" sizes="16x16 32x32 64x64" href="/static/favicon/favicon.ico">
    <link rel="icon" type="image/png" sizes="196x196" href="/static/favicon/favicon-192.png">
    <link rel="icon" type="image/png" sizes="160x160" href="/static/favicon/favicon-160.png">
    <link rel="icon" type="image/png" sizes="96x96" href="/static/favicon/favicon-96.png">
    <link rel="icon" type="image/png" sizes="64x64" href="/static/favicon/favicon-64.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/static/favicon/favicon-32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/static/favicon/favicon-16.png">
    <link rel="apple-touch-icon" href="/static/favicon/favicon-57.png">
    <link rel="apple-touch-icon" sizes="114x114" href="/static/favicon/favicon-114.png">
    <link rel="apple-touch-icon" sizes="72x72" href="/static/favicon/favicon-72.png">
    <link rel="apple-touch-icon" sizes="144x144" href="/static/favicon/favicon-144.png">
    <link rel="apple-touch-icon" sizes="60x60" href="/static/favicon/favicon-60.png">
    <link rel="apple-touch-icon" sizes="120x120" href="/static/favicon/favicon-120.png">
    <link rel="apple-touch-icon" sizes="76x76" href="/static/favicon/favicon-76.png">
    <link rel="apple-touch-icon" sizes="152x152" href="/static/favicon/favicon-152.png">
    <link rel="apple-touch-icon" sizes="180x180" href="//static/favicon/favicon-180.png">
    <meta name="msapplication-TileColor" content="#FFFFFF">
    <meta name="msapplication-TileImage" content="//static/favicon/favicon-144.png">
    <meta name="msapplication-config" content="/static/favicon/browserconfig.xml">
    
    <meta name="keywords" content="Golden, Colorado, CO, expensive, housing, cost, price, statistics, location, places, real estate">
    <meta name="description" content="The average home price in Golden is $522,200. Compared to other places in this area, Golden is a little more expensive, and it is significantly more expensive compared to the US as a whole. It is substantially more expensive even compared to the city of Denver.">
    <meta name="og:description" content="The average home price in Golden is $522,200. Compared to other places in this area, Golden is a little more expensive, and it is significantly more expensive compared to the US as a whole. It is substantially more expensive even compared to the city of Denver.">
    
        <script type="application/ld+json">{"@context": "https://schema.org", "@type": "Article", "author": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "publisher": {"type": "Organization", "name": "Statistipedia", "url": "https://www.statistipedia.org", "logo": "https://www.statistipedia.org/static/img/icons/logo.svg", "sameAs": "https://www.statistipedia.org"}, "datePublished": "2021-07-08T20:18:15.669083", "dateModified": "2021-07-08T20:18:15.669083", "image": "https://storage.googleapis.com/statistipedia/img/maps/png/co/golden.png", "headline": "Is Golden expensive?", "mainEntity": {"@type": "City", "name": "Golden", "sameAs": "https://www.wikidata.org/wiki/Q983543", "containedInPlace": {"@type": "State", "name": "Golden", "sameAs": "https://www.wikidata.org/wiki/Q1261"}}}</script>
    

    <meta name="og:locale" content="en_US" />
</head>

<body>
<header>
    <div class="inner-header">
        <a class="site-title" href="/">
            Statistipedia
        </a>
    </div>
</header>
<main>
<main>
    <article>
        <h1>Is Golden expensive?</h1>
        

<table class="infobox">
    <tr>
        <th colspan="2" class="title">Golden, CO</th>
    </tr>
    <tr>
        <td colspan="2" class="place-image">
            <picture>
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/webp/co/golden.webp" type="image/webp" />
    <source srcset="https://storage.googleapis.com/statistipedia/img/maps/png/co/golden.png" type="image/png" />
    <img src="https://storage.googleapis.com/statistipedia/img/maps/png/co/golden.png" alt="Map of Golden" />
</picture>
        </td>
    </tr>
    <tbody class="main-info">
        <tr>
            <th>State</th>
            <td>
                <a href="/places/us/co">Colorado</a>
            </td>
        </tr>
        <tr>
            <th>Population</th>
            <td>20,693</td>
        </tr>
    </tbody>
    <tr>
        <th colspan="2" class="header">Links</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                
                    <li>
                        <a href="http://www.cityofgolden.net/" target="_blank">Official website</a>
                    </li>
                
                
                    <li>
                        <a href="https://en.wikipedia.org/wiki/Golden,_Colorado" target="_blank" rel="nofollow">
                            Wikipedia article
                        </a>
                    </li>
                
                <li>
                    <a href="https://data.census.gov/cedsci/profile?g=1600000US0830835" target="_blank">
                        Census profile
                    </a>
                </li>
            </ul>
        </td>
    </tr>
    <tr>
        <th colspan="2" class="header">Maps</th>
    </tr>
    <tr>
        <td colspan="2">
            <ul>
                <li><a target="_blank" href="https://www.openstreetmap.org/search?query=Golden%2C%20CO">OpenStreetMaps</a></li>
                <li><a target="_blank" href="https://www.google.com/maps/place/Golden,+CO">Google Maps</a></li>
                <li><a target="_blank" href="https://www.bing.com/maps?q=Golden,+CO">Bing Maps</a></li>
                <li><a target="_blank" href="https://www.mapquest.com/search/result?query=Golden,+CO">MapQuest</a></li>
            </ul>
        </td>
    </tr>
</table>
        
            <p class="lede">The average home price in <a href="/places/us/co/golden">Golden</a> is $522,200.</p>
            <p class="lede">Compared to other places in this area, Golden is a little more expensive, and it is significantly more expensive compared to the US as a whole. It is substantially more expensive even compared to the city of <a href="/places/us/co/denver/expensive?">Denver</a>.</p>
            <div class="figure">
    <figure class="figure-frame">
        <div class="figure-background">
            <div class="viz">
                
                    <div class="neighbor bar"
                         style="width: 100.0%">
                        Genesee
                    </div>
                
                    <div class="location bar"
                         style="width: 71.11534795042898%">
                        Golden
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 70.36633528530574%">
                        Fairmount
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 67.3566662127196%">
                        Applewood
                    </div>
                
                    <div class="city bar"
                         style="width: 53.19351763584366%">
                        Denver
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 49.67996731581098%">
                        Lakewood
                    </div>
                
                    <div class="country bar"
                         style="width: 40.85523627944982%">
                        US average
                    </div>
                
                    <div class="neighbor bar"
                         style="width: 39.31635571292387%">
                        West Pleasant View
                    </div>
                
            </div>
        </div>
        <figcaption>Comparison of average Golden home price to nearby areas</figcaption>
    </figure>
</div>
            <h2>Nearby places</h2><h3>Genesee</h3><p><a href="/places/us/co/genesee/expensive?">Genesee</a> is 4 miles southwest of Golden. The housing prices there are substantially more expensive. The average home cost there is $734,300, which comes out to $212,100 (40%) higher than Golden.</p><h3>Fairmount</h3><p><a href="/places/us/co/fairmount/expensive?">Fairmount</a> is Golden's neighbor to the northeast. Housing prices there are  similar. Its average home value is $516,700, which comes out to $5,500 (1%) lower than Golden.</p><h3>Applewood</h3><p><a href="/places/us/co/applewood/expensive?">Applewood</a> is 2 miles east of Golden. Its real estate prices are somewhat less expensive. The average home value there is $494,600, which comes out to $27,600 (5%) lower than Golden.</p><h3>Lakewood</h3><p>The city of <a href="/places/us/co/lakewood/expensive?">Lakewood</a> is Golden's neighbor to the southeast. Real estate prices there are a bit less expensive. Its average home value is $364,800, which is $157,400 (30%) lower than Golden.</p><h3>West Pleasant View</h3><p><a href="/places/us/co/west-pleasant-view/expensive?">West Pleasant View</a> is Golden's neighbor to the east. Housing prices there are a lot less expensive. Its average home price is $288,700, which works out to $233,500 (44%) lower than Golden.</p>
            <h2>See also</h2>
<ul><li><a href="/places/us/co/wheat-ridge/expensive?">Wheat Ridge</a></li>
<li><a href="/places/us/co/morrison/expensive?">Morrison</a></li>
<li><a href="/places/us/co/dakota-ridge/expensive?">Dakota Ridge</a></li>
<li><a href="/places/us/co/arvada/expensive?">Arvada</a></li>
<li><a href="/places/us/co/evergreen/expensive?">Evergreen</a></li>
<li><a href="/places/us/co/mountain-view/expensive?">Mountain View</a></li>
<li><a href="/places/us/co/edgewater/expensive?">Edgewater</a></li></ul>
        
        <nav class="categories">
            Categories:
            &nbsp;<a href="/places/us">United States</a>
            &nbsp;<span class="divider">|</span>
            &nbsp;<a href="/places/us/co">Colorado</a>
        </nav>
    </article>
</main>
</main>
<footer>
    <div class="inner-footer">
        <a class="site-title" href="/">
            Statistipedia
        </a>
        <div>

        </div>
        <ul>
            <li>© 2021 Statistipedia</li>
            <li><a href="/about">About</a></li>
            <li><a href="/privacy">Privacy</a></li>
        </ul>
    </div>
</footer>
</body>
</html>